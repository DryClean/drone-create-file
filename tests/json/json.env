PLUGIN_IF=true
PLUGIN_BODY=JSON.stringify({body: `$(cat "some report.txt")`,status: /:grin:/g.test(`$(cat "some report.txt")`) ? 'pass': 'fail'}, null, 2)
PLUGIN_OUT=json.out
