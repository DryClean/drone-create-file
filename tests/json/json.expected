{
  "body": "Great job!\nNo linting errors found.\n:grin:",
  "status": "pass"
}
